src/test/java/com/google/api/codegen/testdata/python_doc_message_library.baseline

```diff
@@ -623,7 +623,7 @@ class Book(object):
       author (string): The name of the book author.
       title (string): The title of the book.
       read (bool): Value indicating whether the book has been read.
-      rating (:class:`google.example.library.v1.library_pb2.Book.Rating`): For testing enums.
+      rating (enum from :class:`google.example.library.v1.enums.Rating`): For testing enums.
```

Comment:
I would just say "enum" rather than "enum for". That parallels the Py34 documentation for official enums: https://docs.python.org/3.4/library/enum.html#enum.Enum